packages/react-ui-components/src/components/SchemaForm/SchemaForm.tsx

```diff
@@ -10,14 +12,18 @@ import styles from './SchemaForm.module.scss';
 import Theme from './Theme';
 
 const ThemedForm = withTheme(Theme);
-const { getSubmitButtonOptions } = utils;
-const SCROLL_OFFSET = 10;
+// Custom validator
+const ajvOptionsOverrides: CustomValidatorOptionsType['ajvOptionsOverrides'] = { $data: true };
+const ajvFormatOptions: CustomValidatorOptionsType['ajvFormatOptions'] = {
+	keywords: true //Mandatory for formatMinimum,...
+};
+const defaultValidator = customizeValidator({ ajvOptionsOverrides, ajvFormatOptions });
 
-export interface SchemaForm<T> extends Form<T> {
-	formElement: HTMLDivElement;
-}
+const SCROLL_OFFSET = 10;
```

Comment:
Also, move to a config file.

Is this offset scroll in pixels? 
If so it doesn't seem to comply with our spacing system. 🤷‍♂️

Reply:
Yes it's. I can update it.